### PR TITLE
issue92: fixed KlerosPOC tests

### PR DESCRIPTION
**Pull request by hellwolf**
Changes:

- import 'kleros-interaction/contracts/standard/arbitration/ArbitrableTransaction.sol'
- fix construction parameters order for all occurrences of ArbitrableTransaction

### CODE REVIEW

File: contracts/kleros/POC/KlerosPOCCourt.sol [hunk 1]
```diff
@@ -1,5 +1,6 @@
 pragma solidity ^0.4.15;
 
+import "kleros-interaction/contracts/standard/arbitration/ArbitrableTransaction.sol";
```
Comment (epiqueras) [inline contracts/kleros/POC/KlerosPOCCourt.sol]: Why is this needed here?

Comment (epiqueras) [reply contracts/kleros/POC/KlerosPOCCourt.sol]: Oh, so it can be compiled, because it is from another repo?

Comment (hellwolf) [reply contracts/kleros/POC/KlerosPOCCourt.sol]: yes, it was exactly the reason. It seems that truffle wound't compile it if it is not referenced anywhere in the code.

**Comment by hellwolf** [comment]
abandoned for https://github.com/kleros/kleros/pull/99

**Pull request closed without merging**